apps/frontend/app/(website)/studio/features/FirstLoginExperience/config.tsx

```diff
@@ -4,22 +4,22 @@ const steps = [
   {
     element: "codemod",
     description:
-      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p> Here, you can see an example codemod script that transforms code from a 'Before' state to an 'After' state using jscodeshift.</p><p> You can modify the codemod's code here.</p>",
+      "<p>This section is dedicated to codemods.</p><p>Codemods are tools used to automate code modifications and refactorings. They allow developers to make large-scale changes across a codebase quickly and consistently.</p><p>The codemod editor allows you to see and modify the codemod you are building.</p>",
   },
   {
     element: "before-and-after-panels",
     description:
-      "<p>These panels display the code before and after applying a codemod. The 'Before' panel shows the original code, while the 'After' panel displays the expected output after transformation.</p><p> This visual comparison helps users understand the changes made by the codemod, ensuring the modifications meet the desired outcome.</p>",
+      "<p>The before/after panels allow you to specify how the code should look like before and after transformation.</p><p>To get started, insert your current code into the 'Before' panel, and how you want your code to look like after running the codemod into the 'After' panel.</p>",
```

Comment:
tooooo many words.

"This is your input, in the form of before/after test fixtures and comments describing the transformation logic."

Reply:
maybe as a last step, we can add FAQ or something like this for anyone who wants to dig in deeper?

Reply:
The product should be so intuitive that deep dive is not needed. Anything specific u have in mind? If not let's just keep it as short and simple as possible